packages/btc-wallet/connectors/okx.ts

```diff
@@ -0,0 +1,72 @@
+import { Unisat } from '../unisat'
```

Comment:
Perhaps it's time to rename `Unisat` to something more generic. It feels a weird mix using Unisat here. What do you think?

Reply:
"BaseConnector" and if changes are needed, either OKX or unisat extend it lol